docs/core/testing/unit-testing-best-practices.md

```diff
@@ -75,9 +76,9 @@ purchase.ValidateOrders();
 Assert.True(purchase.CanBeShipped);
 ```
 
-This would be an example of stub being referred to as a mock. In this case, it is a stub. You're just passing in the Order as a means to be able to instantiate `Purchase` (the system under test). The name `MockOrder` is also misleading because again, the order is not a mock.
+The above example would be of stub being referred to as a mock. In this case, it's a stub. You're just passing in the Order as a means to be able to instantiate `Purchase` (the system under test). The name `MockOrder` is also misleading because again, the order isn't a mock.
```

Comment:
I know you're not ready for reviews yet, but I just want to nip this one in the bud in case you make similar changes to other articles. See https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=27392.

Reply:
Noted, thank you! :)